src/main/java/de/tum/in/www1/artemis/domain/exam/ExamUser.java

```diff
@@ -0,0 +1,224 @@
+package de.tum.in.www1.artemis.domain.exam;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.PostLoad;
+import javax.persistence.PostPersist;
+import javax.persistence.PostRemove;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+import javax.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.config.Constants;
+import de.tum.in.www1.artemis.domain.AbstractAuditingEntity;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.service.FilePathService;
+import de.tum.in.www1.artemis.service.FileService;
+
+@Entity
+@Table(name = "exam_user")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class ExamUser extends AbstractAuditingEntity {
+
+    @Transient
+    private final transient FileService fileService = new FileService();
+
+    @Transient
+    private String prevSigningImagePath;
+
+    @Transient
+    private String prevStudentImagePath;
+
+    @Column(name = "actual_room")
+    private String actualRoom;
+
+    @Column(name = "actual_seat")
+    private String actualSeat;
+
+    @Column(name = "planned_room")
+    private String plannedRoom = "not set";
+
+    @Column(name = "planned_seat")
+    private String plannedSeat = "not set";
+
+    @Column(name = "did_check_image")
+    private Boolean didCheckImage = false;
+
+    @Column(name = "did_check_name")
+    private Boolean didCheckName = false;
+
+    @Column(name = "did_check_login")
+    private Boolean didCheckLogin = false;
+
+    @Column(name = "did_check_registration_number")
+    private Boolean didCheckRegistrationNumber = false;
+
+    @Size(max = 100)
+    @Column(name = "signing_image_path", length = 100)
+    private String signingImagePath;
+
+    @Size(max = 100)
+    @Column(name = "student_image_path", length = 100)
+    private String studentImagePath;
```

Comment:
Why is this stored for every ExamUser, wouldn't it be enough to store it once for each User?

Reply:
Users should not see this image, only instructors when making the exam attendance and that's why we need to store for every ExamUser

Reply:
Can you elaborate what you mean by that?

Reply:
We decided to store the image of a user always related to the exam for now and not in the User itself. This might change in the future, but for now it was the best solution